gix/Cargo.toml

```diff
@@ -252,9 +252,9 @@ pack-cache-lru-static = ["gix-pack/pack-cache-lru-static"]
 ## Provide a hash-map based LRU cache whose eviction is based a memory cap calculated from object data.
 pack-cache-lru-dynamic = ["gix-pack/pack-cache-lru-dynamic"]
 
-## Activate other features that maximize performance, like usage of threads, `zlib-ng` and access to caching in object databases.
+## Activate other features that maximize performance, like usage of threads, `zlib-rs` and access to caching in object databases.
 ## Note that some platforms might suffer from compile failures, which is when `max-performance-safe` should be used.
-max-performance = ["max-performance-safe", "zlib-ng", "fast-sha1"]
+max-performance = ["max-performance-safe", "zlib-rs", "fast-sha1"]
```

Comment:
I don't want to push my luck 😅, and would prefer to start by merely making the use of `zlib-rs` easy with the provided feature.

Reply:
Maybe another feature then?

In cargo-binstall we want to get rids of zlib-ng as we already pulled in zlib-rs, having zlib-ng actually makes cross compilation harder

Reply:
@Byron reverted this change